sched/group/group_addrenv.c

```diff
@@ -85,9 +85,11 @@ int group_addrenv(FAR struct tcb_s *tcb)
   FAR struct task_group_s *group;
   FAR struct task_group_s *oldgroup;
   irqstate_t flags;
-  pid_t pid;
+  int cpu;
   int ret;
 
+  cpu = this_cpu();
```

Comment:
need move after line 118, otherwise it may stale if this task migrate to other cpu in the middle of execution.